spec/models/application_record_spec.rb

```diff
@@ -206,56 +238,74 @@
       # technically. gender_eeo_data uses it.
       call
     end
+    it 'calls AR#in_department to filter application records' do
+      relation = ApplicationRecord.all
+      expect(ApplicationRecord).to receive(:in_department)
+        .with(@department.id)
+        .and_return(relation).at_least(:once)
+      # needs to return an activerecord relation. Also
+      # the method :in_department
+      # technically. gender_eeo_data uses it.
+      call
+    end
     it 'returns a hash' do
       expect(call).to be_a Hash
     end
     it 'assigns records to the hash within the correct date range' do
       create :application_record,
              ethnicity: 'Klingon',
-             gender: 'Female'
+             gender: 'Female',
+             position: @position
       Timecop.freeze 2.weeks.ago do
         create :application_record,
                ethnicity: 'Klingon',
-               gender: 'Female'
+               gender: 'Female',
+               position: @position
       end
       expect(call[:all].count).to eql 1
     end
     it 'counts records containing ethnicities not from ethnicity_options' do
       create :application_record,
              ethnicity: 'Betazoid',
-             gender: 'Male'
+             gender: 'Male',
+             position: @position
       expect(call[:ethnicities]).to contain_exactly ['Betazoid', 1],
                                                     ['Klingon', 0]
     end
     it 'calls AR#gender_eeo_data to assign values to genders key in hash' do
       expect(ApplicationRecord).to receive(:gender_eeo_data)
-        .with(@start_date, @end_date)
+        .with(@start_date, @end_date, @department.id)
         .and_return('something')
       call
       expect(call[:genders]).to eql 'something'
     end
     it 'puts all genders and the counts thereof in the hash' do
       create :application_record,
-             gender: 'Female'
+             gender: 'Female',
+             position: @position
```

Comment:
In cases where a bunch of things are the same except for one attribute, I like to put just that attribute on its own line, e.g.:

```ruby
create :application_record, position: @position,
  gender: 'Female'
```